latest/ug/cluster-management/cost-monitoring-kubecost.adoc

```diff
@@ -18,104 +18,16 @@ Kubecost v2 introduces several major new features. <<kubecost-v2,Learn more abou
 
 For more information about Kubecost, see the https://www.ibm.com/docs/en/kubecost/self-hosted/2.x[Kubecost] documentation and <<cost-monitoring-faq, Frequently asked questions>>.
 
+[#kubecost-overview]
+== Install Amazon EKS optimized Kubecost bundle
 
-[#kubecost-addon]
-== Install Kubecost using Amazon EKS Add-ons
+You can use one of the following procedures to install the _Amazon EKS optimized Kubecost bundle_:
 
```

Comment:
as a general note, when you delete lines like` [#kubecost-addon]` you need to look elsewhere in the docs for links to `kubecost-addon`

Reply:
i am happy to do this though :)

Reply:
Nice catch, thanks!

Reply:
Fixed (with squashed commit)